client/api/index.js

```diff
@@ -58,22 +58,16 @@ export default class WCStripeAPI {
 	 * @return {Object} The Stripe Object.
 	 */
 	getStripe() {
-		const {
-			key,
-			locale,
-			isUPEEnabled,
-			paymentMethodsConfig,
-		} = this.options;
+		const { key, locale, isUPEEnabled, paymentMethodsConfig } =
```

Comment:
IDK why this happened and I think the pre-commit hook is not allowing me to push for some reason. I'll fix later.

Reply:
I had a similar issue in the platform checkout repo. The `prettier` rules were invalid for the linter. It started from nowhere. I had to change the linter rules to accept the `prettier` syntax.
Let me know if you figure out something.